app/src/main/java/net/gsantner/markor/activity/MainActivity.java

```diff
@@ -309,7 +309,7 @@ public void onClickFab(final View view) {
 
     private void newItemCallback(final File file) {
         if (file.isFile()) {
-            DocumentActivity.launch(MainActivity.this, file, false, null, null);
+            DocumentActivity.launch(MainActivity.this, file, false, null);
```

Comment:
Removed the option to pass an intent into the launch function. It was used nowhere.